apps/docs/content/components/Section/index.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Section
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/Section/Section.tsx
+storybook: '/brand/storybook/?path=/story/components-section--default'
+description: Use the section component to create a semantic section in the page and group related content together.
+a11yReviewed: false
+---
+
+import {PropTableValues} from '../../../src/components'
+import {
+  SectionBackgroundColors,
+  SectionPaddingVariants,
+} from '@primer/react-brand'
+
+```js
+import {Section} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+The default section wraps its content in a semantic section element and provides default paddings.
+
+```jsx live
+<Section>
+  <SectionIntro align="center">
+    <SectionIntro.Heading size="3">Default section</SectionIntro.Heading>
+    <SectionIntro.Description>
+      This content is wrapped in a Section component.
+    </SectionIntro.Description>
+  </SectionIntro>
+</Section>
+```
+
+### With indepenedent block paddings
+
+```jsx live
+<Section paddingBlockStart="none" paddingBlockEnd="spacious">
+  <SectionIntro align="center">
+    <SectionIntro.Heading size="3">Custom paddings</SectionIntro.Heading>
+    <SectionIntro.Description>
+      This section has a custom padding at the top and bottom.
+    </SectionIntro.Description>
+  </SectionIntro>
+</Section>
+```
+
+### With predefined background color
+
+```jsx live
+<Section backgroundColor="inset">
+  <SectionIntro align="center">
+    <SectionIntro.Heading size="3">Predefined background</SectionIntro.Heading>
+    <SectionIntro.Description>
+      This section has a predefined background color.
+    </SectionIntro.Description>
+  </SectionIntro>
+</Section>
+```
+
+### With custom background color
+
+```jsx live
+<Section backgroundColor="#FFF8F8">
+  <SectionIntro align="center">
+    <SectionIntro.Heading size="3">Custom background</SectionIntro.Heading>
+    <SectionIntro.Description>
+      This section uses a custom background color.
+    </SectionIntro.Description>
+  </SectionIntro>
+</Section>
+```
+
+### With background image
+
+```jsx live
+<Section
+  backgroundImageSrc="https://github.com/user-attachments/assets/068634a3-926d-49b8-b949-0b1e2a204651"
```

Comment:
Could you reupload this into the `primer/brand` repo please? It's currently attached to your user account.